### PR TITLE
Archive command

### PR DESCRIPTION
**Pull request by larnaa**

### CODE REVIEW

**Comment by larnaa** [comment]
@droserasprout this

File: src/bluecoins_cli/cli.py [hunk 1]
```diff
@@ -62,3 +63,19 @@ async def convert(
...
+) -> None:
+    conn = open_copy(ctx.obj['path'])
+
+    cache = QuoteCache(os.path.join(xdg.xdg_cache_home(), 'bluecoins-cli', 'quotes.json'))
+    cache.load()
+
+    create_Archive_account(conn)
+
+    cache.save()
+    print('buy')
```
Comment (droserasprout) [inline src/bluecoins_cli/cli.py:80]: y?😭

File: src/bluecoins_cli/cli.py [hunk 2]
```diff
@@ -62,3 +63,19 @@ async def convert(
...
+@click.pass_context
+async def create_archive(
+    ctx: click.Context
+) -> None:
+    conn = open_copy(ctx.obj['path'])
+
+    cache = QuoteCache(os.path.join(xdg.xdg_cache_home(), 'bluecoins-cli', 'quotes.json'))
+    cache.load()
+
+    create_Archive_account(conn)
```
Comment (droserasprout) [inline src/bluecoins_cli/cli.py:77]: The transaction is not committed. Wrap with `transaction(conn)` as in convert command.

Comment (droserasprout) [inline src/bluecoins_cli/cli.py:77]: The transaction is not committed. Wrap with `transaction(conn)` as in convert command.

File: src/bluecoins_cli/cli.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
 from bluecoins_cli.database import transaction
 from bluecoins_cli.database import update_account
 from bluecoins_cli.database import update_transaction
+from bluecoins_cli.database import create_Archive_account
```
Comment (droserasprout) [inline src/bluecoins_cli/cli.py:17]: Function names must be in snake_case.

**Review by droserasprout — CHANGES_REQUESTED** [review]

File: src/bluecoins_cli/cli.py (hunk 2 above)
Comment (larnaa) [reply src/bluecoins_cli/cli.py]: Wraped.

**Comment by larnaa** [comment]
@droserasprout 
I added a new command for CLI: add a new label.

**Review by droserasprout — APPROVED** [review]

**Pull request merged**